DS/DSAssignment2/CollegeCounselingSystem.java

```diff
@@ -0,0 +1,148 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+class Student {
+	private String name;
+	private int rank;
+
+	public Student(String name, int rank) {
+		this.name = name;
+		this.rank = rank;
+
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public int getRank() {
+		return rank;
+	}
+
+	public void setRank(int rank) {
+		this.rank = rank;
+	}
+}
+
+class College {
+	private String name;
+	private int noOfSeatsAvailable;
+
+	public College(String name, int noOfSeatsAvailable) {
+
+		this.name = name;
+		this.noOfSeatsAvailable = noOfSeatsAvailable;
+	}
+
+	public String getName() {
+		return name;
+	}
+
+	public void setName(String name) {
+		this.name = name;
+	}
+
+	public int getNoOfSeatsAvailable() {
+		return noOfSeatsAvailable;
+	}
+
+	public void setNoOfSeatsAvailable(int noOfSeatsAvailable) {
+		this.noOfSeatsAvailable = noOfSeatsAvailable;
+	}
+
+}
+
+public class CollegeCounselingSystem {
+
+	private static ArrayList<Student> studentList = new ArrayList<Student>();
+	private static ArrayList<College> collegeList = new ArrayList<College>();
+	private static Queue studentQueue;
+
+	public CollegeCounselingSystem() {
+		studentList.add(new Student("udit10", 10));
```

Comment:
ACTION: Why adding students in the constructor, shouldn't this be done in a test class? If in future i want to use this system, i will get 'udit1..udit2...udit3' by default :)